### ISSUE TITLE
Missing argument 2 for Composer\Downloader\FileDownloader::getCacheKey()

### ISSUE THREAD

**Issue by Zae** [composer/composer#4900]
I just started receiving these using the latest version, reverting to an old version solved it.

~~~~php
[ErrorException]                                                                                                                                             
Missing argument 2 for Composer\Downloader\FileDownloader::getCacheKey(), called in phar:///home/releases/20160210151058/api/composer.phar/src/Composer/Downloader/FileDownloader.php on line 188
~~~~

**Comment by xabbuh** [comment]
should be fixed by 8808638ea993633a6f9e5af240bfcf9714b9201a

**Comment by Seldaek** [comment]
Yes indeed please run self-update again
